04/app.js

```diff
@@ -10,17 +11,21 @@ class App extends React.Component {
     }
 
     getInfo = (files) => {
-        this.setState({filesList: files}) // skoro nie robimy usuwania to rozumiem, że dane mają byc nadpisywane 
+        this.setState({filesList: files})  
     }
 
     render() {
+        const copyItems = this.state.filesList.map(el=>{
+            return {...el, id:uuid()}
```

Comment:
Tutaj ciągle jest tworzone nowe id, lepiej to zrobić w linii nr 14

Reply:
Zmienione